prusti/tests/extern_spec/fail/mismatched-sigs.rs

```diff
@@ -0,0 +1,15 @@
+extern crate prusti_contracts;
+use prusti_contracts::*;
+
+#[extern_spec]
+impl<T> std::vec::Vec<T> {
```

Comment:
As a test, this should fail if either of the specifications work (when they shouldn't). Will this test as a whole not fail even if e.g. `push` happens to verify?

Reply:
You are correct in that the test should fail if either of the specifications work. However, it will currently pass even if one happens to verify. I'll split this test case so that both cases are checked for correctly.